lychee-bin/src/formatters/stats/markdown.rs

```diff
@@ -53,6 +56,34 @@ fn stats_table(stats: &ResponseStats) -> String {
         .to_string()
 }
 
+/// Helper function to format single response body as markdown
+///
+/// Optional details get added if available.
+fn markdown_response(response: &ResponseBody) -> Result<String> {
```

Comment:
This function is pretty similar to the default response body formatting at
https://github.com/lycheeverse/lychee/blob/03d28820bbe19de642a11ceccebfbb3e6a58c91e/lychee-lib/src/types/response.rs#L58-L99.
I hope we could refactor that a bit to avoid the code duplication.
Tried a few attempts but I couldn't come up with anything satisfactory. Maybe someone has an idea. If not we can always refactor that later.

Reply:
just leave them like this. Any solution I can think of is over engineering.